md/release-notes.md

```diff
@@ -43,6 +43,7 @@ To be documented
 ### API removals
   
 ## Breaking changes
+* Change in identityAPI to forbidden group name update while new name already exist.
```

Comment:
- Change in identityAPI to _forbidden_ group name

- Since it will be rewritten next release anyway, might be good to add the method name here.